submittable/component-library
# Fix esm bundle

---
---

### Testing
1. npm run build && npm pack
2. go to FE and install the tzg file
3. npm i ../component-library/submittable-csr-component-library-1.2.0.tgz
4. start your app
5. update one of the components to retrieve it from component-library
6. everything should work

### Demo (recommended)
Screenshots or videos of the UI added in this PR
